src/test/java/uk/gov/register/presentation/functional/FunctionalTest.java

```diff
@@ -41,6 +45,26 @@ public void shouldConsumeMessageFromKafkaAndShowAsLatest() throws Exception {
 
     }
 
+    @Test
+    public void appSupportsCORS() {
```

Comment:
The test confirms that app provides the CORS support. It has to be a functional test, doesn't matter  how it is implemented. I am agree that functional test suite should provide quick feedback regardless of number of tests in it.